.travis.yml

```diff
@@ -7,6 +7,11 @@ python:
   - 3.6
   - pypy
   - pypy3.5
+matrix:
+  include:
+  - python: 3.7
+    sudo: required
```

Comment:
Out of curiosity, why do we need to set `sudo` to `required` for Python 3.7?

Reply:
It's all coming from here https://github.com/travis-ci/travis-ci/issues/9069; the `sudo: required` AFAIK is necessary to let Travis spin another virtual machine on Google Compute Engine with a different Linux distribution (`xenial`).
Some info here a well https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system.

Reply:
Thanks! Can we add a comment with a link to https://github.com/travis-ci/travis-ci/issues/9069 so we don't accidentally delete this in the future?